tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -442,6 +448,26 @@ void CreateLineProbesFile(ExplorationTestDescription testDescription)
         }
     }
 
+    static bool TryLoadAssembly(string testAssemblyPath, [NotNullWhen(true)] out Assembly assembly)
+    {
+        try
+        {
+            assembly = Assembly.LoadFile(testAssemblyPath);
+            return true;
+        }
+        catch (BadImageFormatException e)
```

Comment:
Not a blocker, but this is now causing compiler warnings on `master` 😅 
> `The variable 'e' is declared but never used`

Reply:
yeah I saw that... hang in there I'll push a little PR

Reply:
https://github.com/DataDog/dd-trace-dotnet/pull/6546